### PR TITLE
UI updated along with UX improvements

### PR DESCRIPTION
**Pull request by shankarsengalani**
UI Scrolling on public data explorer page fixed

---

This change is [Reviewable](https://reviewable.io/reviews/maidsafe/safe_examples/101)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @krishnaIndia

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: demo_app/app/backend/directory_helper.js [hunk 1]
```diff
@@ -8,20 +8,24 @@ export let computeDirectorySize = function(localPath) {
...
-  for (var i in contents) {
-    tempPath = localPath + '/' + contents[i];
-    stat = fs.statSync(tempPath);
-    if (stat.isDirectory()) {
-      size += computeDirectorySize(tempPath);
-    } else {
-      if (env.isFileUploadSizeRestricted && stat.size > env.maxFileUploadSize) {
-        throw new Error('File more than ' + (env.maxFileUploadSize / 1000000) + ' Mb can not be uploaded');
+  try {
+    let contents = fs.readdirSync(localPath);
```
Comment (dirvine) [inline demo_app/app/backend/directory_helper.js]: I wonder why the use of Sync calls here and not async, is there a performance hit here?

Comment (krishnaIndia) [inline demo_app/app/backend/directory_helper.js]: This operations does block the UI, in the sense, we don't allow users to do any other action in the mean time.But definitely would be better to make it async. There are few more places which uses blocking calls like this using `fs`, which can be updated in a separate PR. Will that be fine?
